Packs/ThreatConnect/Integrations/ThreatConnectV3/ThreatConnectV3.yml

```diff
@@ -2133,6 +2147,773 @@ script:
     - contextPath: TC.Group.ID
       description: The ID of the group.
       type: string
+  - arguments:
+      - default: true
+        description: The name of the victim.
+        name: name
+        required: true
+      - description: The nationality of the victim.
+        name: nationality
+      - description: The organization of the victim.
+        name: org
+      - description: The sub-organization of the victim.
+        name: sub_org
+      - auto: PREDEFINED
+        description: A comma-separated list of the security labels to apply to the victim.
+        name: security_labels
```

Comment:
as this defined as CSV, it better to define `isArray:true`
please update all the CSV args.
in addition, It's a hole in our UI that predefined values can't selected as CSV values 
so user need to put them manually, so i assume defined them as predefined is just for "know" the options.

Reply:
will add to all relevant places